### PR TITLE
Settings Sync not supported for remote

### PR DESCRIPTION
**Pull request by ntrogh**
Fixes https://github.com/microsoft/vscode-docs/issues/7221

### CODE REVIEW

File: docs/editor/extension-marketplace.md
[the remark this answers is not in the document]
Comment (ntrogh) [reply docs/editor/extension-marketplace.md:1]: I'll take it out.

**Review by sandy081 — APPROVED** [review]

**Comment by sandy081** [comment]
Thanks for the PR

**Comment by thernstig** [comment]
@ntrogh @sandy081 I am not sure the changes are correct. The note says:

> **Note**: VS Code does not synchronize settings to or from a [remote](/docs/remote/remote-overview.md) window, such as when you're connected to SSH, a development container (devcontainer), or WSL.

It does synchronize settings. The settings (Settings, Keyboard Shorcuts, User Snippets etc.) are not dependent on local/remote. It is just the **extensions** that do not sync.

**Comment by sandy081** [comment]
@thernstig Thanks for pointing it out.

@ntrogh My bad, I missed it. Thanks for updating.

**Pull request merged**